tests/functional/test_thermal_heuristic_three_clusters.py

```diff
@@ -0,0 +1,276 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+from math import ceil
+from pathlib import Path
+
+import ortools.linear_solver.pywraplp as pywraplp
+import pytest
+
+from andromede.libs.standard import (
+    BALANCE_PORT_TYPE,
+    DEMAND_MODEL,
+    NODE_BALANCE_MODEL,
+    SPILLAGE_MODEL,
+    UNSUPPLIED_ENERGY_MODEL,
+)
+from andromede.thermal_heuristic.cluster_parameter import compute_delta
+from andromede.thermal_heuristic.data import ExpectedOutput, ExpectedOutputIndexes
+from andromede.thermal_heuristic.model import (
+    AccurateModelBuilder,
+    FastModelBuilder,
+    HeuristicAccurateModelBuilder,
+    HeuristicFastModelBuilder,
+    Model,
+)
+from andromede.thermal_heuristic.problem import (
+    SolvingParameters,
+    ThermalProblemBuilder,
+    TimeScenarioHourParameter,
+    WeekScenarioIndex,
+)
+from tests.functional.libs.lib_thermal_heuristic import THERMAL_CLUSTER_MODEL_MILP
+
+
+@pytest.fixture
+def solver_parameters() -> pywraplp.MPSolverParameters:
+    parameters = pywraplp.MPSolverParameters()
+    parameters.SetIntegerParam(parameters.PRESOLVE, parameters.PRESOLVE_OFF)
+    parameters.SetIntegerParam(parameters.SCALING, 0)
+    parameters.SetDoubleParam(parameters.RELATIVE_MIP_GAP, 1e-5)
+    return parameters
+
+
+@pytest.fixture
+def data_path() -> str:
+    return "data/thermal_heuristic_three_clusters"
+
+
+@pytest.fixture
+def models() -> list[Model]:
```

Comment:
This fixture seems common to many test, is it possible to define it in a conftest file and import it ?